pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -70,7 +70,7 @@ PARSER_END(VfParser)
 | <#XMLNAME: (<ALPHA_CHAR> | "_" | ":") (<IDENTIFIER_CHAR>)* >
 | <#QUOTED_STRING_NO_BREAKS: ( "'" ( ~["'", "\r", "\n"] )* "'" )
                     | ( "\"" ( ~["\"", "\r", "\n"] )* "\"" ) >
-| <#QUOTED_STRING: ( "'" ( ~["'"] )* "'" ) | ( "\"" ( ~["\""] )* "\"" ) >
+| <#QUOTED_STRING: ( "'" ( ~["'"] )* "'" ) | ( "\"" ( ~["\""] | "\\\"" )* "\"" ) >
```

Comment:
@jsotuyod This would be superfluous, VisualForce rewrites all double quotes in the source code into single quotes for the resulting page. Putting something like <abc ab=' \' '>abc</abc> causes a compilation error and <abc ab=' \' >abc</abc> does not.